omod/src/main/java/org/openmrs/module/sync2/page/controller/Sync2PageController.java

```diff
@@ -0,0 +1,41 @@
+package org.openmrs.module.sync2.page.controller;
+
+import org.apache.commons.lang.StringUtils;
+import org.openmrs.module.sync2.api.utils.SyncUtils;
+import org.openmrs.module.uicommons.util.InfoErrorMessageUtil;
+import org.openmrs.ui.framework.UiUtils;
+import org.openmrs.ui.framework.page.PageModel;
+import org.springframework.stereotype.Controller;
+
+import javax.servlet.http.HttpSession;
+
+/**
+ * The Sync 2 page controller.
+ */
+@Controller
+public class Sync2PageController {
+
+	private static final String VALID_PARENT_INSTANCE_URI = "validParentInstanceUri";
+
+	private static final String PARENT_URI_ERROR = "sync2.configuration.parentUrl.empty";
+
+	/**
+	 *
+	 * Method using when page is refreshing.
```

Comment:
Generally we should omit using method/class words in javadoc, because it is obvious that it is a method. It is a good pattern to start with verb in 3rd person, e.g. 
'Returns Patient object', 'Sets the tool tip text', 'Registers ...'

Please try to use this convention

Reply:
I didn't know. Thanks a lot. I changed this Javadoc.